wicket-util/src/main/java/org/apache/wicket/util/encoding/UrlDecoder.java

```diff
@@ -16,25 +16,25 @@
  */
 package org.apache.wicket.util.encoding;
 
+import java.io.ByteArrayOutputStream;
 import java.io.UnsupportedEncodingException;
 import java.nio.charset.Charset;
 import java.nio.charset.IllegalCharsetNameException;
 import java.nio.charset.UnsupportedCharsetException;
 
 import org.apache.wicket.util.lang.Args;
-import org.apache.wicket.util.string.Strings;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Adapted from java.net.URLDecoder, but defines instances for query string decoding versus URL path
+ * Adapted from Spring's UriUtils, but defines instances for query string decoding versus URL path
  * component decoding.
  * <p/>
  * The difference is important because a space is encoded as a + in a query string, but this is a
  * valid value in a path component (and is therefore not decode back to a space).
- * 
- * @author Doug Donohoe
- * @see java.net.URLDecoder
+ *
+ * @author Thomas Heigl
+ * @see org.springframework.web.util.UriUtils

```

Comment:
I think this breaks the Javadoc build because wicket-util does not depend on spring-web.
Please the first line of the javadoc with something like "Adapted from Spring Framework's UrilUtils class"

Reply:
Done